index.js

```diff
@@ -16,11 +16,38 @@ mongoose
   })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
+
     // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+    //return Recipe.deleteMany()
+    
   })
-  .then(() => {
+  .then(async(req, res) => {
     // Run your code here, after you have insured that the connection was made
+    //agregando una nueva receta
+    const myNewRecipe = {
```

Comment:
Buen trabajo Sinuhe, note que estabas trabajando con asyn entonces el manejo de errores es diferente que con .then y .catch agregue una alternativa 😉
```suggestion
const myNewRecipe = {
  title: "Lemon Pie",
  level: "Amateur Chef",
  ingredients: [
    "1/2 cup light brown sugar",
    "1 large egg",
    "2 Philadelphia Cheese",
    "1 1/4 Marias Cookies",
    "15 lemon slices",
  ],
  cuisine: "French",
  dishType: "dessert",
  image:
    "https://paydelimon.org/wp-content/uploads/2017/12/10.-relleno-de-lemon-pie.jpg",
  duration: 50,
  creator: "Chef Sinuhe",
};
//const recipeCreated = await Recipe.create(myNewRecipe);
//console.log(recipeCreated.title);
//agregando todas las recetas a partir del Recipe y obteniendo el data.json

const handleData = async () => {
  const allRecipes = await Recipe.insertMany(data);
  allRecipes.forEach((element) => {
    console.log(element.title); //no puedo imprimir los titles aun
  });
};

try {
  handleData();
} catch (error) {
  console.error("Error connecting to the database", error);
}
```